tests/generic_config_updater/test_acl.py

```diff
@@ -0,0 +1,360 @@
+import logging
+import pytest
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.config_reload import config_reload
+from tests.generic_config_updater.gu_utils import apply_patch, expect_op_success, expect_res_success, expect_op_failure
+from tests.generic_config_updater.gu_utils import generate_tmpfile, delete_tmpfile
+
+pytestmark = [
+    pytest.mark.topology('t0'),
+]
+
+logger = logging.getLogger(__name__)
+
+@pytest.fixture(scope="module", autouse=True)
+def setup_env(duthosts, rand_one_dut_hostname, cfg_facts):
+    """
+    Setup/teardown fixture for acl config
+    Args:
+        duthosts: list of DUTs.
+        rand_selected_dut: The fixture returns a randomly selected DuT.
+        cfg_facts: config facts for selected DUT
+    """
+    duthost = duthosts[rand_one_dut_hostname]
+
+    config_tmpfile = generate_tmpfile(duthost)
+    logger.info("config_tmpfile {} Backing up config_db.json".format(config_tmpfile))
+    duthost.shell("sudo cp /etc/sonic/config_db.json {}".format(config_tmpfile))
+
+    # Cleanup acl config
+    duthost.shell('sonic-db-cli CONFIG_DB keys "ACL_RULE|*" | xargs sonic-db-cli CONFIG_DB del',
```

Comment:
>xargs [](http://example.com/codeflow?start=62&length=5)

Did you test with empty ACL_RULE table? I guess you need the option `--no-run-if-empty`.

Reply:
I tested with empty and the command failed. So I added `module_ignore_errors=True` to get that error ignored.
Seems yours is the better way to handle an empty table. I will remove `module_ignore_error`